Intermittent ChannelFull test failure

e.g. https://github.com/django/channels_redis/runs/1328351504?check_suite_focus=true#step:6:105

This is an artefact: not consistent on run or version. 

Need to make the test more reliable.

Going to note which test(s) fail(s). (If it's always the same one...)

https://github.com/django/channels_redis/runs/2919887893?check_suite_focus=true
```
FAILED tests/test_core.py::test_receive_cancel
```

https://github.com/django/channels_redis/pull/262/checks?check_run_id=3016793723

```
FAILED tests/test_core.py::test_receive_cancel - channels.exceptions.ChannelFull
```

https://github.com/django/channels_redis/pull/262/checks?check_run_id=3017259693

```
FAILED tests/test_core.py::test_receive_cancel - channels.exceptions.ChannelFull
```


Looks like doubling the capacity in `test_receive_cancel` might be required here.

--- a/tests/test_core.py
+++ b/tests/test_core.py
@@ -436,7 +436,7 @@ async def test_receive_cancel(channel_layer):
     """
     Makes sure we can cancel a receive without blocking
     """
-    channel_layer = RedisChannelLayer(capacity=10)
+    channel_layer = RedisChannelLayer(capacity=20)
     channel = await channel_layer.new_channel()
     delay = 0
     while delay < 0.01:

--- a/tests/test_sentinel.py
+++ b/tests/test_sentinel.py
@@ -439,7 +439,7 @@ async def test_receive_cancel(channel_layer):
     """
     Makes sure we can cancel a receive without blocking
     """
-    channel_layer = RedisChannelLayer(capacity=10)
+    channel_layer = RedisChannelLayer(capacity=20)
     channel = await channel_layer.new_channel()
     delay = 0
     while delay < 0.01:

